package.xml

```diff
@@ -0,0 +1,93 @@
+                    <file name="ddtrace.h" role="src" />
+                    <file name="debug.h" role="src" />
+                    <file name="dispatch.c" role="src" />
+                    <file name="dispatch.h" role="src" />
+                    <file name="dispatch_compat.h" role="src" />
+                    <file name="dispatch_compat_php5.c" role="src" />
+                    <file name="dispatch_compat_php5.h" role="src" />
+                    <file name="dispatch_compat_php7.c" role="src" />
+                    <file name="dispatch_setup.c" role="src" />
+                    <file name="version.h" role="src" />
+                </dir>
+            </dir>
+            <dir name="tests">
+                <dir name="ext">
+                    <file name="access_modifier_method_access_hook.phpt" role="test" />
+                    <file name="access_modifier_property_access_hook.phpt" role="test" />
+                    <file name="case_insensitive_method_hook.phpt" role="test" />
+                    <file name="closure_accessing_outside_variables.phpt" role="test" />
+                    <file name="closure_set_inside_object_methods.phpt" role="test" />
+                    <file name="method_returning_array.phpt" role="test" />
+                    <file name="multiple_instrumentations.phpt" role="test" />
+                    <file name="namespaces.phpt" role="test" />
+                    <file name="overriding_construct.phpt" role="test" />
+                    <file name="overriding_method_defined_in_parent.phpt" role="test" />
+                    <file name="private_method_hook.phpt" role="test" />
+                    <file name="protected_method_hook.phpt" role="test" />
+                    <file name="recursion.phpt" role="test" />
+                    <file name="return_value_passed.phpt" role="test" />
+                    <file name="simple_function_hook.phpt" role="test" />
+                    <file name="simple_method_hook.phpt" role="test" />
+                    <file name="throw_exception.phpt" role="test" />
+                    <file name="variable_length_parameter_list.phpt" role="test" />
+                    <file name="with_params_function_hook.phpt" role="test" />
+                    <file name="with_params_method_hook.phpt" role="test" />
+                </dir>
+            </dir>
+            <file name="CHANGELOG.md" role="doc" />
+            <file name="config.m4" role="src" />
+            <file name="LICENSE" role="doc" />
+            <file name="README.md" role="doc" />
+        </dir>
+    </contents>
+    <dependencies>
+        <required>
+            <php>
+                <min>5.6</min>
+            </php>
+            <pearinstaller>
+                <min>1.4.0</min>
+            </pearinstaller>
+            <extension>
+                <name>json</name>
+            </extension>
+        </required>
+    </dependencies>
+    <providesextension>ddtrace</providesextension>
```

Comment:
I checked a couple of extensions and seems that typically `<name/>` and `<providesextension/>` match. I am wondering what are the cons/issues we face if we provide the name `datadog_trace` also here.

Reply:
some other extension have different name, this is not a blocker (e.g. pecl_http / http, apcu_bc / apc, ...)

Reply:
Thanks @remicollet